src/steps/account/account-field-equals.ts

```diff
@@ -31,10 +31,11 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
     optionality: FieldDefinition.Optionality.OPTIONAL,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
+    description: 'Check Logic (set to, not set to, containing, not containing, greater than, less than, be set, or not be set)',
```

Comment:
Looks like bad copy pasta from Campaign Member step applied to all the others.  This (and others) should still read `be, not be, contain, not contain, be greater than, be less than, be set, or not be set)`